pubspec.yaml

```diff
@@ -84,6 +84,9 @@ dependencies:
   #filesize
   filesize: ^1.0.4
 
+  # share
+  share: ^0.6.5+4
```

Comment:
I think from now, we should not use `^`, it is risk for us too much.

Reply:
agree